recipes/libhdfs3/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.2.31" %}
+
+package:
+  name: libhdfs3
+  version: "{{ version }}"
+
+source:
+    git_url: https://github.com/PivotalRD/libhdfs3.git
+    git_tag: v{{ version }}
+
+build:
+  number: 0
+  # This package is a dependency of hdfs3, which is primarily used with HDFS on Linux.
```

Comment:
No need for this comment. I am curious about OS X though. Did you try to enable it?